### ISSUE TITLE
Hexadecimal Keyboard example

### ISSUE THREAD

**Issue by dpadula** [wix/react-native-ui-lib#1342]
Hello I wanted to ask you, if were possible of course, can you create a tiny functional component example of mini hexadecimal keyboard? I can't make work the example indicated by @M-i-k-e-l. At least not as I expect. The keyboard that shows is the traditional on Android not a custom one.
Can you give me extra help in this task? I really want to this works in my app. Maybe to show as a real app example.
Can I leave you an example of how it has to look ? 
Thanks in advance

**Comment by M-i-k-e-l** [comment]
Hi @dpadula,

I've created [a gist](https://gist.github.com/M-i-k-e-l/e6e34c8c35197feeba963cf19cc69ea6) to show how a button of a hex letter can be implemented and used.
It has a modified version of `demoKeyboards.js` and `KeyboardInputViewScreen.js` which can be found in this project.
After you modify the files, search for the `KeyboardAccessoryView` screen in our demo app and press on the custom keyboard (the icon with four squares), you'll see a button `A`, press it and see the interaction.
* I did not handle deletion etc.

Hope this helps you,
Regards
